frontend/public/components/edit-yaml.jsx

```diff
@@ -569,6 +570,15 @@ export const EditYAML_ = connect(stateToProps)(
           window.dispatchEvent(new Event('sidebar_toggle'));
         };
 
+        toggleHover = () => {
+          this.setState(
```

Comment:
Can we make this a user preference so that the value is preserved? And we can consider adding it to the preferences page as well.

Reply:
@spadgett i was thinking i wasn't sure which way to go on this...I'm open to adding a user pref. My struggle was that once a pref is set will the user remember and just have the hover forever disabled? But I think you're suggesting that this would still sit on the edit yaml page and  remind the user of the current setting, yet still set a user pref, and also have that setting in the user prefs page. Ok, interesting....didn't think of that!

Reply:
agreed this should be a checkbox. 
Question about the user pref page - would the user toggle it there and only see the state of that on the YAML view? Or would they be able to toggle it from either place? OR would the user pref be the default and then per YAML they can decide to enable disable it?

Reply:
@spadgett I added the user preference code and as per our discussion on monday, I believe we agreed not to add this to the user pref page. @matthewcarleton I used a checkbox instead of the link button.